fdbclient/include/fdbclient/GetEncryptCipherKeys.actor.h

```diff
@@ -108,10 +138,13 @@ Future<std::unordered_map<EncryptCipherDomainId, Reference<BlobCipherKey>>> getL
 			    domain.first /*domainId*/, domain.second /*domainName*/, request.arena);
 		}
 	}
+	g_blobCipherKeyCacheMetrics.cacheHit += cipherKeys.size();
+	g_blobCipherKeyCacheMetrics.cacheMiss += domains.size() - cipherKeys.size();
 	if (request.encryptDomainInfos.empty()) {
 		return cipherKeys;
 	}
 	// Fetch any uncached cipher keys.
+	state double startTime = now();
```

Comment:
for better understanding; I see three time measurement methods; do we know major difference between them and usescases they fit the best? 

```
	double now() const override { return currentTime; };
	double timer() override { return ::timer(); };
	double timer_monotonic() override { return ::timer_monotonic(); };
```

For latency profiling I see `timer_monotonic()` being used around the code snippet that needs to be profiled 

`
double elapsed = -timer_monotonic();
.......<code>
elapsed += timer_monotonic();
` 

Any reason we aren't following the same semantics?

Reply:
`now` is a variable updated every time the run loop goes around from `timer_monotonic`, and so it is very cheap to access. It is typically pretty accurate, but sometimes it can be imprecise, especially if you have a long running task that isn't yielding.

`timer` and `timer_monotonic` actually get the current time from the OS. `timer` gets a non-monotonic timestamp that represents what the OS thinks the current time is, and `timer_monotonic` is a timer that always increases and is probably better for measuring durations.

Both `timer` and `timer_monotonic` are more expensive than `now`, so you should only use them if you need the extra precision they would provide.

Reply:
updated to use `timer_monotonic()` to measure cpu time of crypto ops, and `now()` to measure EKP access.